### ISSUE TITLE
hfpdtext does not compile

### ISSUE THREAD

**Issue by marado** [heinervdm/nohands#9]
Seems that this tool was forgotten during the various refactoring processes...

**Comment by heinervdm** [comment]
This was never compiled in my enviroment...
And it seems that this can't compile since a very long time (at least since 2009) and non of my changes broke it.
No idea if I can make it compile again...
Do you know what it does?

**Comment by marado** [comment]
Yeah, I actually went backwards in the commits and it is broken since 2008/2009. 
According to README, at least, it's a command line client (as a replacement to the available GUI's).
